src/System.Management.Automation/engine/remoting/commands/InvokeCommandCommand.cs

```diff
@@ -263,6 +263,72 @@ public override SwitchParameter UseSSL
             }
         }
 
+        private sealed class ArgumentToPSVersionTransformationAttribute : ArgumentToVersionTransformationAttribute
```

Comment:
@Thomas-Yu , @daxian-dbw   For the record, I don't think it was necessary to split the attribute/conversion classes into two separate implementations.  Soon,`Set-StrictMode` will also take the `off` value for strict mode version, which means merging these two classes back together.  In the mean time, there was no harm in having one set of attribute/conversion implementations.  Now when `Set-StrictMode` is updated to also support `off` this work will have to be refactored yet again.

@Thomas-Yu  Please create an Issue in PowerShell repo that adds `off` to the `Set-StrictMode -Version` parameter.  The committee would like to see it work the same as for `Invoke-Command` here for consistency, although it is not necessary to include the changes in this PR.